build.gradle

```diff
@@ -13,7 +13,7 @@ import java.text.SimpleDateFormat
 
 buildscript {
     ext {
-        version_gradle_tools = "3.2.1"
+        version_gradle_tools = '3.3.2'
```

Comment:
Guess these two changes were automatically done by the `first start popup`?

You can as well revert changes on this file